src/plugins/network.ts

```diff
@@ -165,12 +177,32 @@ export class NetworkManager {
     if (!record) {
       return;
     }
-    Object.assign(record, {
-      status: response.status,
-      response,
-      responseReceivedTime: timestamp,
-      transferSize: response.encodedDataLength,
-    });
+
+    record.status = response.status;
+    record.responseReceivedTime = timestamp;
+    record.transferSize = response.encodedDataLength;
+    record.response = {
+      url: response.url,
+      statusCode: response.status,
```

Comment:
The Uptime UI expects this to be `status`, not `statusCode`. Status is missing from waterfall chart in this version of the synthetics agent.

Reply:
I thought we used the status from the top level not inside the `response`. You are right - https://github.com/elastic/kibana/blob/affbed7a161ae5b41d4dde8264b597b7c21f453e/x-pack/plugins/uptime/server/lib/requests/get_network_events.ts#L70

I will remove the top level fields that are unused in this case.